admin-dev/themes/new-theme/js/components/form-submit-button.js

```diff
@@ -60,11 +60,32 @@ export default class FormSubmitButton {
         return;
       }
 
+      let method = 'POST';
+      let addInput = null;
+
+      if ($btn.data('method')) {
+        const btnMethod = $btn.data('method');
+        const isGetOrPostMethod = ['GET', 'POST'].includes(btnMethod);
+        method = isGetOrPostMethod ? method : 'POST';
```

Comment:
It seems like the value of the **method** variable can only be 'POST'. Or is there something I don't get ?